index.html

```diff
@@ -24,8 +24,127 @@ <h1>I’m Mariane Glad to see you!</h1>
+                <h2>Tonic</h2>
+                <ul> 
+                    <li>CANOPY</li>
+                    <li>⬤</li>
+                    <li>BackEnd Dev</li>
+                    <li>⬤</li>
+                    <li>2015</li>
+                </ul>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+            <button>See Project</button>
+            </div>
+        <div class="cards card4">
+            <img src="./assets/Snapshoot3.png" alt="">
+            <div class="card1">
+                <h2>Multi-Post Stories</h2>
+                <ul> 
+                    <li>CANOPY</li>
+                    <li>⬤</li>
+                    <li>BackEnd Dev</li>
+                    <li>⬤</li>
+                    <li>2015</li>
+                </ul>
+                </div>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li>
+                </ul>
+                <button>See Project</button>
+            </div>
+    </section>
+    <section class="about">
+        <div class="about-social">
+        <h2>About Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="social-title-1">LET’S CONNECT</p>
+        <ul>
+            <li class="social" ><a href=""><img src="assets/twitter.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Linkedinicon.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector2.png" alt=""></a></li>
+            <li class="social" ><a href=""><img src="assets/Vector3.png" alt=""></a></li>
+        </ul>
+        <button>Get my resume</button>
+        </div>
+        <ul class="about-list">
+            <li>
+                <h2>Languages</h2>
+                <p>˅</p>
+            </li>
+            <li><img src="./assets/js.png" alt="">
+            <p>javaScript</p>
+            </li>
+            <li><img src="./assets/html.png" alt="">
+            <p>html</p>
+            </li>
+            <li><img src="./assets/css.png" alt="">
+            <p>css</p>
+            </li>
+            <li>
+                <h2>Frameworks</h2>
+                <p>></p>
+            </li>
+            <li>
+                <h2>Skills</h2>
+                <p>></p>
+            </li>
+        </ul>
+    </section>
```

Comment:
- [ ] Please increase font weight for all these headings to match your design with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/185662000-fa6f755d-d26e-4a03-99ce-574853909f91.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/185662049-79efc1e3-538c-4171-afb1-3fec7a20d0f1.png)